tests/Kinepolis/KinepolisDateParserTest.php

```diff
@@ -0,0 +1,972 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Kinepolis;
+
+use CultuurNet\UDB3\Model\ValueObject\Calendar\BookingAvailability;
+use CultuurNet\UDB3\Model\ValueObject\Calendar\BookingAvailabilityType;
+use CultuurNet\UDB3\Model\ValueObject\Calendar\DateRange;
+use CultuurNet\UDB3\Model\ValueObject\Calendar\Status;
+use CultuurNet\UDB3\Model\ValueObject\Calendar\StatusType;
+use CultuurNet\UDB3\Model\ValueObject\Calendar\SubEvent;
+use PHPUnit\Framework\TestCase;
+
+final class KinepolisDateParserTest extends TestCase
+{
+    private KinepolisDateParser $dateParser;
+
+    public function setUp(): void
+    {
+        $this->dateParser = new KinepolisDateParser();
+    }
+
+    /**
+     * @test
+     * @dataProvider dateDataProvider
+     */
+    public function it_converts_dates_from_kinepolis(array $data, int $length, array $result): void
+    {
+        $this->assertEquals(
+            $this->dateParser->processDates($data, $length),
+            $result
+        );
+    }
+
+    public function dateDataProvider(): array
```

Comment:
Very nice to see these detailed tests, but maybe the movie arrays can be placed inside simple PHP files that return an array?